README.md

```diff
@@ -1,39 +1,60 @@
 # Beaker
 
-Beaker is an acceptance testing harness for Puppet PE and other Puppet Projects.  It can also be used as a virtual machine provisioner - setting up machines, running any configuration on those machines and then exiting.
+Beaker is a test harness focused on acceptance testing via interactions
+between multiple (virtual) machines. It provides platform abstraction between
+different Systems Under Test (SUTs), and it can also be used as a virtual machine
+provisioner - setting up machines, running any commands on those machines
+and then exiting.
```

Comment:
I'm into the [oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/), are you?